package.json

```diff
@@ -2,21 +2,39 @@
   "name": "prettierx",
   "version": "0.19.0-01-update-branch",
   "prettier-version": "2.4.0",
-  "description": "prettierx 0.19.0-01-update-branch (...)",
+  "description": "prettierx 0.19.0-01-update-branch (...) a less opinionated code formatter fork of Prettier",
   "bin": "./bin/prettierx.js",
+  "repository": "brodybits/prettierx",
+  "homepage": "https://github.com/brodybits/prettierx",
   "license": "MIT",
   "main": "./index.js",
-  "browser": "./standalone.js",
-  "unpkg": "./standalone.js",
+  "// prettierx STANDALONE TODO NOTES": "CURRENTLY NOT SUPPORTED",
+  "// browser": "./standalone.js",
+  "// unpkg": "./standalone.js",
```

Comment:
TODO:

- [ ] clean up before publishing release
- [ ] find a way to offer "standalone" version of prettierX (may need to add `standalone.js` back to `files` list below)